silabs-multiprotocol/translations/en.yaml

```diff
@@ -39,4 +39,5 @@ configuration:
       Use OpenThread Border Router firewall to block unnecessary traffic.
 network:
   9999/tcp: EmberZNet EZSP/ASH port
+  8080/tcp: OpenThread Web port
```

Comment:
Seems to be unused?

Reply:
Right, I don't need that when allocating automatically. Will remove those.